Client/Frontend/Browser/Onboarding/OnboardingRewardsAgreementViewController.swift

```diff
@@ -20,27 +47,335 @@ class OnboardingRewardsAgreementViewController: OnboardingViewController {
     override func viewDidLoad() {
         super.viewDidLoad()
         
-        contentView.turnOnButton.addTarget(self, action: #selector(onTurnOn), for: .touchUpInside)
+        view.backgroundColor = #colorLiteral(red: 0.1176470588, green: 0.1254901961, blue: 0.1607843137, alpha: 1)
```

Comment:
what color is this, can we use some value we already have?

Reply:
This is a color we dont haveut I added this color to UX struct and used rgb initializer https://github.com/brave/brave-ios/pull/4508/commits/858063434f530cde5c6e9ff8394401225f85ef09